### PR TITLE
[NUI] Fix svace defects

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###

 Fix svace defects

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by AdunFang — APPROVED** [review]
It is OK.

**Comment by xuelian-bai** [comment]
looks good to me

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by elishateng — APPROVED** [review]
LGTM

**Review by wb0716 — APPROVED** [review]
Good for me

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by AdunFang — APPROVED** [review]
It is OK.

File: src/Tizen.NUI/src/internal/EXaml/Action/GetObjectByPropertyAction.cs [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void Init()
 
         public void OnActive()
         {
-            if (null != childOp)
+            if (null != childOp && null != (childOp.ValueList[0] as Instance))
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/EXaml/Action/GetObjectByPropertyAction.cs:64]: Hello, how about change this as like this?

`if (null != childOp && childOp.ValueList[0] is Instance valueList0)`
and 
`int instanceIndex = valueList0.Index;`

Comment (zhouleonlei) [reply src/Tizen.NUI/src/internal/EXaml/Action/GetObjectByPropertyAction.cs]: > Hello, how about change this as like this?
> 
> `if (null != childOp && childOp.ValueList[0] is Instance valueList0)`
> and
> `int instanceIndex = valueList0.Index;`

Fixed

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**